client/my-sites/email/email-management/index.jsx

```diff
@@ -0,0 +1,269 @@
+				),
+			};
+		}
+
+		const emailForwardingAction = {
+			secondaryAction: translate( 'Add email forwarding' ),
+			secondaryActionURL: emailManagementForwarding( selectedSiteSlug, selectedDomainName ),
+		};
+
+		if (
+			selectedDomain &&
+			isMappedDomain( selectedDomain ) &&
+			! isMappedDomainWithWpcomNameservers( selectedDomain )
+		) {
+			return {
+				title: translate( 'Use the powerful features of G Suite on this domain' ),
+				line: translate(
+					'To enable G Suite on %(domain)s, please configure it to use WordPress.com name servers.',
+					{ args: { domain: selectedDomainName } }
+				),
+				action: translate( 'How to change your name servers' ),
+				actionURL: localizeUrl(
+					'https://wordpress.com/support/domains/map-existing-domain/#change-your-domains-name-servers'
+				),
+				actionTarget: '_blank',
+				...emailForwardingAction,
+			};
+		}
+
+		if ( selectedDomainName ) {
+			return {
+				title: translate( 'G Suite is not supported on this domain' ),
+				line: translate( 'Only domains registered with WordPress.com are eligible for G Suite.' ),
+				...emailForwardingAction,
+			};
+		}
+
+		return {
+			title: translate( 'Enable powerful email features.' ),
+			line: translate(
+				'To set up email forwarding, G Suite, and other email ' +
+					'services for your site, upgrade your site’s web address ' +
+					'to a professional custom domain.'
+			),
+		};
+	}
+
+	googleAppsUsersCard() {
+		const { domains, gsuiteUsers, selectedDomainName } = this.props;
+
+		return (
+			<GSuiteUsersCard
+				domains={ domains }
+				gsuiteUsers={ gsuiteUsers }
+				selectedDomainName={ selectedDomainName }
+			/>
+		);
+	}
+
+	goToEditOrList = () => {
+		const { selectedDomainName, selectedSiteSlug, currentRoute, previousRoute } = this.props;
+		const domainPath = domainManagementEdit( selectedSiteSlug, selectedDomainName, currentRoute );
+		const emailPath = emailManagement( selectedSiteSlug );
+
+		if (
+			selectedDomainName &&
+			( previousRoute.startsWith( domainPath ) ||
+				currentRoute.startsWith( emailPath ) ||
+				isUnderEmailManagementAll( currentRoute ) )
+		) {
+			page( domainPath );
+		} else {
+			page( domainManagementList( selectedSiteSlug, currentRoute ) );
+		}
+	};
+}
+
+export default connect( ( state ) => {
+	const selectedSiteId = getSelectedSiteId( state );
+
+	return {
+		currentRoute: getCurrentRoute( state ),
+		canManageSite: canCurrentUser( state, selectedSiteId, 'manage_options' ),
+		domains: getDomainsBySiteId( state, selectedSiteId ),
+		gsuiteUsers: getGSuiteUsers( state, selectedSiteId ),
+		hasGSuiteUsersLoaded: hasLoadedGSuiteUsers( state, selectedSiteId ),
+		hasSiteDomainsLoaded: hasLoadedSiteDomains( state, selectedSiteId ),
+		previousRoute: getPreviousRoute( state ),
+		selectedSiteId,
+		selectedSiteSlug: getSelectedSiteSlug( state ),
+	};
+} )( localize( EmailManagement ) );
```

Comment:
This file [already exists](https://github.com/Automattic/wp-calypso/blob/16a727db49f88d12fd1a13b24c6d010544c89cfb/client/my-sites/email/email-management/index.jsx), so I'm not sure why it shows up in this pull request.

Reply:
Ups, quite strange. It might be a problem while updating the branch.